packages/components/Callout/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@fluentui-react-native/callout",
+  "version": "0.1.0",
+  "description": "A cross-platform opinionated Fluent Callout component",
+  "main": "lib/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "scripts": {
+    "build": "just-scripts build",
+    "just": "just-scripts",
+    "clean": "just-scripts clean",
+    "update-snapshots": "just-scripts jest -u",
+    "verify-api": "just-scripts verify-api-extractor",
+    "update-api": "just-scripts update-api-extractor"
+  },
+  "dependencies": {
+    "@uifabricshared/foundation-compose": "^1.5.9",
+    "@fluentui-react-native/adapters": "^0.1.0",
+    "@fluentui-react-native/interactive-hooks": "^0.1.0",
+    "@fluentui-react-native/tokens": "^0.1.0"
```

Comment:
Please make sure the dependencies are right. Some of these look wrong, every dependency on something published out of this repo should be on the latest version. You should see warnings in the console during bundle that mention this. e.g. tokens, adapters, interactive-hooks all look out of date. You can hover on the text in vs code to see the latest version.
- Taken from what Alicia commented on my PR regarding Checkbox. Applies here as well

Reply:
checked, all good :)

Reply:
@PPatBoyd your statement may no longer be true when you pull from master.  I know adapters is for sure at 1.1, for example